examples/react-apollo/cypress/e2e/3-sign-in/email-password.cy.ts

```diff
@@ -0,0 +1,24 @@
+import { faker } from '@faker-js/faker'
+
+context('Sign in with email+password', () => {
+  const email = faker.internet.email()
+  const password = faker.internet.password()
+
+  it('should sign-up', () => {
+    cy.signUpEmailPassword(email, password)
+    cy.contains('Verification email sent').should('be.visible')
+    cy.confirmEmail(email)
+  })
+
+  it('should sign-in with email and password', () => {
```

Comment:
Tests shouldn't depend on each other. Skipping the first one would break this test.